cmd/policy_run.go

```diff
@@ -47,6 +48,10 @@ func init() {
 	flags := policyRunCmd.Flags()
 	flags.StringVar(&outputDir, "output-dir", "", "Generates a new file for each policy at the given dir with the output")
 	flags.BoolVar(&noResults, "no-results", false, "Do not show policies results")
+	flags.Bool("disable-fetch-check", false, "Disable checking if a respective fetch happened before running policies")
```

Comment:
nit: This seems a bit out of style with other flags (which are bound to variables and pass their values as arguments to `RunPolicies`). As a consequence introduces dependency on viper package into pkg/policy package. Maybe we could combine those flags into a structure and still pass it as an argument..

Reply:
Discussed on slack :)